dhcpv4/dhcpv4.go

```diff
@@ -574,7 +580,8 @@ func (d *DHCPv4) RootPath() string {
 //
 // The Bootfile Name option is described by RFC 2132, Section 9.5.
 func (d *DHCPv4) BootFileNameOption() string {
-	return GetString(OptionBootfileName, d.Options)
+	name := GetString(OptionBootfileName, d.Options)
+	return strings.TrimRight(name, "\x00")
```

Comment:
that's sad.. I wonder if we should notify the user of this or silently swallow the null bytes. Thank you null-terminated strings? :(

Reply:
As RFC2132 states "Options containing NVT ASCII data SHOULD NOT include a trailing NULL; however, the receiver of such options MUST be prepared to delete trailing nulls if they exist." :-)

Reply:
sounds good :)